src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_strategy_selection_test.py

```diff
@@ -58,4 +62,84 @@ class TestStrategySelectionPatchless(unittest.TestCase):
   def test_strategy_pool_generator(self):
     """Ensures that a call to generate_strategy_pool does not yield an
     exception. Deterministic behaviors are tested in the previous test."""
+    strategy_selection.generate_random_strategy_pool()
+
+
+@test_utils.with_cloud_emulators('datastore')
+class TestMultiArmedBanditStrategySelectionPatch(unittest.TestCase):
+  """Tests whether a multi armed bandit strategy pool is properly
+  generated according to the specified distribution."""
+
+  def setUp(self):
+    """Put data in the local ndb table the tests to query from and set
+    bandit selection environment variable."""
+    test_helpers.patch_environ(self)
+    os.environ['USE_BANDIT_STRATEGY_SELECTION'] = 'True'
+
+    data = []
+
+    strategy1 = data_types.FuzzStrategyProbability()
+    strategy1.strategy_name = 'fork,corpus_subset,recommended_dict,'
+    strategy1.probability = 0.33
+    data.append(strategy1)
+
+    strategy2 = data_types.FuzzStrategyProbability()
+    strategy2.strategy_name = ('random_max_len,corpus_mutations_ml_rnn,'
+                               'value_profile,recommended_dict,')
+    strategy2.probability = .34
+    data.append(strategy2)
+
+    strategy3 = data_types.FuzzStrategyProbability()
+    strategy3.strategy_name = ('corpus_mutations_radamsa,'
+                               'random_max_len,corpus_subset,')
+    strategy3.probability = .33
+    data.append(strategy3)
+    ndb.put_multi(data)
+
+  def test_multi_armed_bandit_strategy_pool(self):
+    """Ensures a call to the multi armed bandit strategy
+    selection function doesn't yield an exception."""
```

Comment:
Why not to verify that the resulting `pool` object is correct?

Reply:
This includes a non-deterministic call to pool, so we simply test whether or not a pool was successfully generated. A separate deterministic test is also included to verify that the resulting pool object was correct.

Reply:
Ack